mitiq/interface/conversions.py

```diff
@@ -20,6 +20,51 @@ class CircuitConversionError(Exception):
     pass
 
 
+register_to_dict: Dict[Tuple[str, str], Callable[[Any], Circuit]]
+try:
+    register_to_dict
+except NameError:
+    register_to_dict = {}
+
+register_from_dict: Dict[Tuple[str, str], Callable[[Circuit], Any]]
+try:
+    register_from_dict
+except NameError:
+    register_from_dict = {}
```

Comment:
Can you explain this code a little bit? I have a feeling it has something to do with not wanting to overwrite an existing conversion dictionary, but I'm not totally sure.

Reply:
That's correct. Initially the explicit `global` sufficed, but this one became necessary at some point (I can't remember exactly why), so in #1864 I left this `try/catch` but removed `global`.